assets/sass/components/idea-hub/_googlesitekit-idea-hub-dashboard-ideas-widget.scss

```diff
@@ -67,4 +71,153 @@
 			}
 		}
 	}
+
```

Comment:
We also need to fix tabs issue on very narrow screens because if I enable mobile view in the storybook, I can't select the third tab: 
![google github io_site-kit-wp_storybook_pull_3382__path=_story_modules-idea-hub-widgets-dashboardideaswidget--ready](https://user-images.githubusercontent.com/873049/119027179-b65e1800-b9ae-11eb-8fca-fd8d05b888cf.png)

Reply:
@eugene-manuilov This was reported here: https://github.com/google/site-kit-wp/issues/3174 but basically the mobile view in Storybook is erroneous. The actual width of the content area us `220px` due to some extra `100px` padding on the sides in Storybook. This is how it should look on `320px` screens:
![image](https://user-images.githubusercontent.com/31818/119130521-64b69b80-ba49-11eb-9b64-0a318e5aa56e.png)
And I don't think we need to cater for screens lower than `320px`. Besides, the user can swipe to reveal more tab items.

Reply:
Got it. Thanks, @asvinb!